Ansible/group_vars/all.sample

```diff
@@ -748,6 +751,7 @@ def_vmware_templates:
     esxi_template: "na"
     vc_template: "na"
 def_vc_service_offering: "VC server"
+def_vc_nested_service_offering: "Nested VC Server"
```

Comment:
Is this service offering for RAM >=14 GB (12GB for vcenter and 1-2 GB for ESXi?)

Reply:
Yes - this one is 15GB RAM x 4CPUs